scikeras/wrappers.py

```diff
@@ -422,8 +432,14 @@ def _build_keras_model(self):
 
     def _ensure_compiled_model(self) -> None:
         # compile model if user gave us an un-compiled model
-        if not (hasattr(self.model_, "loss") and hasattr(self.model_, "optimizer")):
-            self.model_.compile(**self._get_compile_kwargs())
+        if not (
+            getattr(self.model_, "loss", None)
+            and getattr(self.model_, "optimizer", None)
```

Comment:
Why is this change necessary?

Reply:
Good catch. This change isn't really necessary. I'll remove it.

Reply:
Reverted